api/src/main/java/org/openmrs/module/mirebalais/require/RequireUtil.java

```diff
@@ -71,6 +71,10 @@ public static String patientYoungerThan(int age) {
         return "patient.person.age <= " + age;
     }
 
+    public static String patientYoungerThanMonths(int ageMonths) {
+        return "(new Date(visit.startDatetime) - new Date(patient.person.birthdate)) / (1000 * 60 * 60 * 24 * 30.4) <= " + ageMonths;
```

Comment:
Can we try:  moment(visit.startDatetime).diff(moment(visit.patient.person.birthdate), 'months')

Reply:
Worth a shot, but hese require rules are written in JavaScript to allow them to portable and evaluated either client or server side...  in Java, they are evaluated using the Nashorn engine within the Appframework, and assumedly moment is not available there (though we could potentially see about loading it see: AppFrameworkServiceImpl:99

Reply:
Yeah, tried that, `moment` is not available.